azext_edge/edge/_help.py

```diff
@@ -1219,3 +1219,81 @@ def load_iotops_help():
             --sa-resource-id $STORAGE_ACCOUNT_RESOURCE_ID --sa-container myschemacontainer
             -l westus2 --desc 'Contoso factory X1 schemas' --display-name 'Contoso X1' --tags env=prod
     """
+
+    helps[
+        "iot ops connector"
+    ] = """
+        type: group
+        short-summary: Connector management.
+    """
+
+    helps[
+        "iot ops connector opcua"
+    ] = """
+        type: group
+        short-summary: OPC UA connector management.
+        long-summary: |
+          The connector for OPC UA enables your industrial OPC UA environment to ingress data into
+          your local workloads running on a Kubernetes cluster, and into your cloud workloads.
+          See the following resource for more info https://aka.ms/overview-connector-opcua-broker
+    """
+
+    helps[
+        "iot ops connector opcua trust"
+    ] = """
+        type: group
+        short-summary: Manage trusted certificates for the OPC UA Broker.
+        long-summary: |
+          The trusted certificate list contains the certificates of all the OPC UA servers that the connector
+          for OPC UA trusts. If the connector for OPC UA trusts a certificate authority, it automatically trusts
+          any server that has a valid application instance certificate signed by the certificate authority.
+          See the following resource for more info https://aka.ms/overview-opcua-broker-certificates-management
+    """
+
+    helps[
+        "iot ops connector opcua trust add"
+    ] = """
+        type: command
+        short-summary: Add a trusted certificate to the OPC UA Broker's trusted certificate list.
+        long-summary: |
+            The certificate file extension must be .der or .crt.
```

Comment:
get ready for people trying to use pem files...

also see if crt is the same as cert for your case.

Reply:
"From the docs, PEM must be in a .crt file. (the user can convert if necessary)" quoting from here https://microsoft.sharepoint.com/:w:/t/DigitalOperationsExperiences386/EUegQTElRzRMl9bY3_HyPNoBssVovqX5bmacDjW4DC78Eg?e=kqGSUc